MainModule/Server/Core/Functions.luau

```diff
@@ -1319,7 +1319,7 @@ return function(Vargs, GetEnv)
 				if #tab>=num then
 					break
 				elseif #tab>=num-1 then
-					table.insert(tab,string.sub(msg,#str+3,#msg))
+					table.insert(tab,string.sub(msg,#str+#tab+1,#msg))
```

Comment:
This still breaks command arguments. I it works with
```diff
- table.insert(tab,string.sub(msg,#str+#tab+1,#msg))
+ table.insert(tab,string.sub(msg,#str+#tab,#msg))
```

but even with that the original issue with command quotes persists.

Reply:
Hence why im trying something different